### PR TITLE
Rename vipps files

### PR DESCRIPTION
**Pull request by SvenErikPy**

### CODE REVIEW

File: checkout-api.md [hunk 1]
```diff
@@ -60,11 +60,11 @@ and you will not get improvements and bug fixes we do in the SDK.
...
+With *Checkout Direct* you can easily implement an express checkout experience directly from a single product without going through a shopping cart. *Checkout Direct*  decouples you from needing to embed the iFrame and lets us handle everything at `checkout.vipps.no` before returning the customer to your shop after a payment is finished. To use *Checkout Direct* , follow the [System integration guidelines](#system-integration-guidelines) and make sure you pick [Alternative 2](#alternative-2-checkout-direct---we-handle-the-checkout-and-redirect-the-user-back-to-you) under [Step 2: Displaying the session to the user](#step-2-displaying-the-session).
```
Comment (SvenErikPy) [inline checkout-api.md:67]: Unsure about this link

File: checkout-api.md [hunk 2]
```diff
@@ -241,7 +241,7 @@ Load the frontend SDK in the `<head>` section of the merchant website.
 
 #### Alternative 1: Classic implementation where Checkout is embedded in an iFrame on your site
 
-This is the standard flow where *Checkout* is embedded on your site typically when a customer checks out a shopping cart. For direct checkout of a single item, see [Checkout Direct](#alternative-2-vipps-checkout-direct---we-handle-the-checkout-and-redirect-the-user-back-to-you). The two alternatives can work side by side in combination on your site.
+This is the standard flow where *Checkout* is embedded on your site typically when a customer checks out a shopping cart. For direct checkout of a single item, see [Checkout Direct](#alternative-2-checkout-direct---we-handle-the-checkout-and-redirect-the-user-back-to-you). The two alternatives can work side by side in combination on your site.
```
Comment (SvenErikPy) [inline checkout-api.md:244]: Same with alternative 2 link here

**Comment by rebekaburnett** [comment]
I can't review this now, but I am ok with it. I've tested it in docusaurus.

I also added a redirect so people will be directed to the new location.

**Comment by SvenErikPy** [comment]
Did the "alternative 2" links work for you?

File: checkout-api.md (hunk 1 above)
Comment (rebekaburnett) [reply checkout-api.md]: yes, this works

File: checkout-api.md (hunk 2 above)
Comment (rebekaburnett) [reply checkout-api.md]: Yes, this also works

**Review by eirikv — APPROVED** [review]

**Pull request merged**